charts/mimir-distributed/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-version: 0.1.6
+version: 2.0.0
```

Comment:
oh, I like keeping the chart version in line with the mimir version, but what happens if the chart requires a breaking change that does not require a corresponding app version update?

Reply:
I think enterprise customers need 2.0.0 for continuity from 1.8.1 + this has breaking changes